src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue

```diff
@@ -203,6 +203,14 @@ export default {
 				return moderator1 ? -1 : 1
 			}
 
+			if (p1inCall) {
+				return participant1.attendeePermissions < participant2.attendeePermissions ? 1 : -1
+			}
+
+			if (p1inCall && participant1.attendeePermissions === participant2.attendeePermissions) {
+				return participant1.inCall < participant2.inCall ? 1 : -1
+			}
```

Comment:
The inCall attribute is already sorted by in line ~179

I think I would do it all the time (independent from inCall or not)

```suggestion
			if (participant1.attendeePermissions !== participant2.attendeePermissions) {
				return participant1.attendeePermissions < participant2.attendeePermissions ? 1 : -1
			}
```

 but only for moderators (other users dont see permissions):
* So include the participant mixin
* check with code like this if the current user is a moderator:
https://github.com/nextcloud/spreed/blob/b06c7f24886e6773219aa60d93e27ee1bb9f59d8/src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue#L614-L616

Reply:
I think already edit in the correct place (right sidebar), but I need to update new logic for admin view, other guys will not change. right?

Reply:
> but I need to update new logic for admin view, other guys will not change. right?

yes